### ISSUE TITLE
Improve design of make chaff popup

### ISSUE THREAD

**Issue by banaagravrommel** [bleachbit/bleachbit#1187]
**Describe the bug**
Improve design of make chaff popup. It should be readable and pleasing to the user's perspective.

**To Reproduce**

1. Open the application
2. Click BleachBit Icon on the top right
3. Select Make Chaff

**Expected behavior**
Popup should be readable and pleasing to the user's perspective.

**Screenshots**
https://ibb.co/xhMr8Zn

**Desktop (please complete the following information):**

- OS: Windows 10
- BleachBit version 4.3.0

**Additional context**

**Comment by az0** [comment]
I agree that the dialog should be improved, though it is not a top priority now. The dialog was maybe not changed since it was added in BleachBit 3.0.

I uploaded your image in case ibb.co later deletes it.

![github_1187_chaff_ui](https://user-images.githubusercontent.com/933680/122653474-ec073580-d101-11eb-9f4c-294c571dfb7c.png)

**Comment by az0** [comment]
6975ba5823492130385207b6f7fd232144441a8a improved display of the chaff GUI, and it was released in 6.0.0
